src/evalfunc.c

```diff
@@ -1152,7 +1150,7 @@ static argcheck_T arg1_len[] = {arg_len1};
 static argcheck_T arg3_libcall[] = {arg_string, arg_string, arg_string_or_nr};
 static argcheck_T arg14_maparg[] = {arg_string, arg_string, arg_bool, arg_bool};
 static argcheck_T arg2_filter[] = {arg_list_or_dict_or_blob_or_string_mod, arg_filter_func};
-static argcheck_T arg2_instanceof[] = {arg_object, arg_class_or_list};
+static argcheck_T arg219_instanceof[] = {arg_object, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class, arg_class};
```

Comment:
This way of specifying the variable length argument types looks messy.  Is there any other way to simplify this?

Many of the internal functions evolved over time by adding additional optional arguments and in most cases Dicts were used for these arguments.  If we specify the maximum number of argument types like this, then we cannot add a different argument type in the future to instanceof().